clang/include/clang/Driver/Options.td

```diff
@@ -3750,6 +3750,11 @@ def fsycl_preserve_device_nonsemantic_metadata : Flag<["-"], "fsycl-preserve-dev
 def ftarget_compile_fast : Flag<["-"], "ftarget-compile-fast">,
   Visibility<[ClangOption, CLOption, DXCOption]>, HelpText<"Experimental feature: Reduce target "
   "compilation time, with potential runtime performance trade-off.">;
+def ftarget_device_link : Flag<["-"], "ftarget-device-link">,
+  Visibility<[ClangOption, CLOption, DXCOption]>, HelpText<"Perform device "
```

Comment:
What is DXCOption?

Reply:
These are options that would be available to the DXC Compiler (DirectX Shading or something like that).  This string of visibility values basically is the equivalent to the old `CoreOption` designator.